tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -1086,54 +1086,4 @@ protected function assertContactHeader(NodeElement $rendered_element, string $ti
     $this->assertEquals($title, $rendered_header->getText());
   }
 
-  /**
-   * Asserts rendering of Contact entity using Details view mode.
-   *
-   * @param \Behat\Mink\Element\NodeElement $element
-   *   Rendered element.
-   * @param string $name
-   *   Name of the Contact entity.
-   */
-  protected function assertContactDetailsRender(NodeElement $element, string $name): void {
```

Comment:
As we are not removing the oe_details configuration that this assertion asserts, we shouldn't remove this assertion either.